GASLanguageProcessor/ToAstVisitor.cs

```diff
@@ -123,11 +123,10 @@ public override AstNode VisitAssignment(GASParser.AssignmentContext context)
 
     public override AstNode VisitGroupTerm(GASParser.GroupTermContext context)
     {
-        var statements = ToCompound(context.statement()
-            .Select(c => c.Accept(this))
-            .ToList());
         var expression = context.expression().Accept(this) as Expression;
-
+        
+        Statement? statements = context.statement() == null ? ToCompound(context.statement().Select(c => c.Accept(this)).ToList()) : null;
```

Comment:
Statement? statements =ToCompound(context.statement()?.Select(c => c.Accept(this)).ToList());

Reply:
ping pong :)
![pingpong](https://github.com/AndeTaar/GASLanguageProcessor/assets/113096859/f4a79e96-8e23-416e-9997-2e882d84b8dd)